### DISCUSSION TITLE
How to get `.conf` syntax hightlighting.

### DISCUSSION THREAD

**Discussion by ghost** [LazyVim/LazyVim#3020] [Q&A] [answered]
I can't find out how to get syntax highlighting for .conf files which are part of my Hyprland setup. Has anyone else got this to work?

**Comment by kik4444** [comment, marked as answer]
Add this plugin to install a hyprlang parser for syntax highlighting.
```lua
  {
    "nvim-treesitter/nvim-treesitter",
    opts = {
      ensure_installed = {
        "hyprlang",
      },
    },
  },
```

Then add this to your `lua/config/options.lua` to identify which files to treat as hyprlang
```lua

vim.filetype.add({
  pattern = { [".*/hypr/.*%.conf"] = "hyprlang" },
})
```

**Reply by ghost** [reply]
Thanks so much!
